apps/mobile/src/navigation/linking/index.js

```diff
@@ -140,7 +140,7 @@ export const routingConfig = {
   '/:context(groups)/:groupSlug/:streamType(proposals)':                  `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
   '/:context(groups)/:groupSlug/:streamType(requests-and-offers)':        `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
   '/:context(groups)/:groupSlug/:streamType(resources)':                  `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
-  '/:context(groups)/:groupSlug':                                         `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab`,
+  '/:context(groups)/:groupSlug':                                         `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Chat Room`,
```

Comment:
You may already be in this, but I am planning to still zoom out and handle the top level like this. Given that we have a homeWidget / homeURL I wasn't expecting this to be pinned down at the level of linking, but I guess it works for now. We may revisit?

Reply:
Yeah, I was trying to pin down whether context switching was happening from links, so this change is incidental. I'll revert

Reply:
Ah yes, no, it is not. That is a part of the plumbing which remains. I will do that part shortly. That's what I mean by the top end of the linking/routing :)